src/Domain.Schema/Attributes/Examples/ExampleAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using NJsonSchema.Annotations;
+
+namespace OMP.Connector.Domain.Schema.Attributes.Examples
+{
+    public abstract class ExampleAttribute : JsonSchemaExtensionDataAttribute
```

Comment:
What is the purpose of the examples?

Reply:
I have no idea. Was part of the code base. The namespance suggest it has no real value but because it is part of the schema project for serialization I am not 100% what or if it will have effects. We can not pick up reference issues on this library on compile time (not on all items)

Reply:
Ok, let's assume we will clean up this at a later stage.
The focus is just to get the things to work.